test/functional/get-test-error.js

```diff
@@ -30,28 +19,36 @@ function sanitizeError (err) {
         .join(' ');
 }
 
-module.exports = function getTestError (testReport, browsersInfo) {
-    var testError = '';
+// NOTE: This method should return dictionary or array object depending on the test result. If testing was against
+// several browsers and same error was occured in all of them, there should be an array with this error. In other
+// case, if tests were against one browser, there should be the array with errors too, regardless of their count.
+// Dictionary will be returned, if we got different errors in several browsers. In this case browser aliases will
+// be keys, and values will be arrays as described above.
+module.exports = function getTestError (testReport, browsers) {
```

Comment:
This method should return either a dictionary or array object with errors depending on the number of browsers and errors. The possible variants:
 
If tests run in one browser, the method returns an array with errors regardless of the number of errors.
 
If tests run in several browsers and the same error occurs in all of them, the method returns an array as well. 
 
If different errors occur in several browsers, a dictionary object is returned. In this case, browser aliases are keys, and values are arrays of errors.